File: FinalProject/Models/Vault.cs
namespace FinalProject.Models;
public class Vault
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public string Img { get; set; }
    public Boolean IsPrivate { get; set; }
    public string CreatorId { get; set; }
    public Account Creator { get; set; }

}

File: FinalProject.client/src/models/Vault.js
import { Account } from "./Account.js"

export class Vault {
    constructor(data) {

        this.id = data.id
        this.name = data.name
        this.description = data.description
        this.img = data.img
        this.isPrivate = data.isPrivate
        this.creatorId = data.creatorId
        // this.Creator = new Account(data.Creator)
    }
}
